### PR TITLE
Add error page 

### LINKED ISSUE #29: [Impl] Implement the Error page.
Implement the not found page where the user enters an invalid link.

### PR DESCRIPTION
**Pull request by sanduninduwara**

### CODE REVIEW

File: src/pages/error-page.js [hunk 1]
```diff
@@ -1,8 +1,81 @@
-import { Typography } from '@mui/material';
+import { Typography } from "@mui/material";
+import { Box } from "@mui/system";
+import { Button, TextField, Link } from "@mui/material";
+import * as React from "react";
```
Comment (Malith-19) [inline src/pages/error-page.js]: we can remove this line right?

File: src/pages/error-page.js [hunk 2]
```diff
@@ -1,8 +1,81 @@
...
+          Page Not Found
+        </Typography>
+        {/* <TextField
+          id="standard-basic"
+          label="Index Number:"
+          variant="standard"
+          size="small"
+          sx={textFieldStyle}
+          onChange={(e) => handleInputChange(e)}
+        /> */}
```
Comment (Malith-19) [inline src/pages/error-page.js]: Let's remove the unnecessary lines.

File: src/pages/schedule-page.js [hunk 3]
```diff
@@ -100,7 +100,7 @@ const data = [
     { id: 16, module: 'Professional Practice', moduleCode: 'CS1456', hall: 'B202', studentCount: '40 users', time: '10:30 AM' },
 ];
 
-const rowsPerPage = 5;
+const rowsPerPage = 7;
```
Comment (Malith-19) [inline src/pages/schedule-page.js:103]: Any specific reason to change this since this is not related to error page?

Comment (sanduninduwara) [reply src/pages/schedule-page.js:103]: yes, 7 will be the right size to fill the page

File: src/pages/error-page.js [hunk 4]
```diff
@@ -1,8 +1,69 @@
-import { Typography } from '@mui/material';
+import { Typography } from "@mui/material";
+import { Box } from "@mui/system";
+import { Button } from "@mui/material";
```
Comment (Malith-19) [inline src/pages/error-page.js]: can we merge these typography and button import like `import {Typography, Button} from '@mui/material';`

**Review by Malith-19 — APPROVED** [review]
